docs/Other SDL Features/Getting In-Car Microphone Audio/index.md

```diff
@@ -32,17 +34,36 @@ let audioPassThru = SDLPerformAudioPassThru(initialPrompt: "<#A speech prompt wh
 
 sdlManager.send(audioPassThru)
 ```
+!@
+
+@![android,javaSE,javaEE]
+```java
+PerformAudioPassThru performAPT = new PerformAudioPassThru();
+performAPT.setAudioPassThruDisplayText1("Ask me \"What's the weather?\"");
+performAPT.setAudioPassThruDisplayText2("or \"What's 1 + 2?\"");
+
+performAPT.setInitialPrompt(TTSChunkFactory.createSimpleTTSChunks("Ask me What's the weather? or What's 1 plus 2?"));
+performAPT.setSamplingRate(SamplingRate._22KHZ);
+performAPT.setMaxDuration(7000);
+performAPT.setBitsPerSample(BitsPerSample._16_BIT);
+performAPT.setAudioType(AudioType.PCM);
+performAPT.setMuteAudio(false);
+
+sdlManager.sendRPC(performAPT);
+```
+!@
 
 ###### Ford HMI
 ![Ford Audio Pass Thru](assets/Ford_AudioPassThruPrompt.png)
 
 ### Gathering Audio Data
-SDL provides audio data as fast as it can gather it, and sends it to the developer in chunks. In order to retrieve this audio data, the developer must add a handler to the `SDLPerformAudioPassThru`.
+@![iOS]SDL provides audio data as fast as it can gather it, and sends it to the developer in chunks. In order to retrieve this audio data, the developer must add a handler to the `SDLPerformAudioPassThru`.!@ @![android,javaSE,javaEE] Before starting audio capture, the app has to subscribe to AudioPassThru notification. SDL provides audio data as fast as it can gather it, and sends it to the developer in chunks. In order to retrieve this audio data, observe the OnAudioPassThru notification.!@
```

Comment:
```suggestion
SDL provides audio data as fast as it can gather it, and sends it to the developer in chunks. In order to retrieve this audio data, the developer must @![iOS]add a handler to the `SDLPerformAudioPassThru`.!@ @![android,javaSE,javaEE]observe the `OnAudioPassThru` notification.!@
```

Reply:
changed